src/components/Editor/index.js

```diff
@@ -305,32 +303,22 @@ class Editor extends PureComponent<Props, State> {
   };
 
   onSearchAgain = (_, e) => {
-    const { query, searchModifiers } = this.props;
-    const { editor: { codeMirror } } = this.state.editor;
-    const ctx = { ed: this.state.editor, cm: codeMirror };
-
-    const direction = e.shiftKey ? "prev" : "next";
-    traverseResults(e, ctx, query, direction, searchModifiers.toJS());
+    this.props.traverseResults(e.shiftKey, this.state.editor);
   };
 
   openMenu(event) {
-    const { setContextMenu } = this.props;
-
     if (event.target.classList.contains("CodeMirror-linenumber")) {
-      return setContextMenu("Gutter", event);
+      return this.props.setContextMenu("Gutter", event);
     }
 
-    return setContextMenu("Editor", event);
+    return this.props.setContextMenu("Editor", event);
   }
 
   onGutterClick = (cm, line, gutter, ev) => {
     const {
       selectedSource,
-      toggleBreakpoint,
```

Comment:
why are we removing these? they seem to still be used the same way as before

Reply:
Ah, forgot to roll that back. Will take care of it. Sorry for the confusion.